src/controllers/wallet.js

```diff
@@ -0,0 +1,56 @@
+var Validator = require('validator');
+var Restify = require('restify');
+
+var WalletModel = require('../models/wallet');
+var Config = require('../config');
+
+var wallet = {
```

Comment:
Perhaps we should rename this to `WalletController` if that's what we're calling it when we require it? :bike: